server/data/helpers/refreshrHelper.js

```diff
@@ -2,32 +2,55 @@ const db = require('../../config/dbConfig');
 
 module.exports = {
 	getRefreshrs: async (id) => {
```

Comment:
So this will return all refreshrs if called with no id, and one refreshr if an id is passed? That's clean